Readme.md

```diff
@@ -0,0 +1,29 @@
+# DigitalBank
+Digital Bank is an example online digital banking application for exploratory purposes.
+## Getting Started
+* Clone [this repository](https://github.com/digisic/digitalbank-gen-one)
+## Prerequisites
+* Install [Java SDK](https://openjdk.java.net/)
+* Install [Node.js](https://nodejs.org/download/release/v12.21.0/)
+* Install [Visual Studio Code](https://code.visualstudio.com/download)
+## Executing VSCode
+* Create a new folder and add it to the workspace
+* In a new terminal run: npm init 
+* Install WebdriverIO: npm install webdriverio --save-dev
+* Install the client: npm install @wdio/cli
+* Create the configuration file: npx wdio config
+* Install Chai: npm install chai --save-dev
+* Install Chai-WebdriverIO: npm install chai-webdriverio --save-dev
+## Executing Tests
+* Change the file wdio.conf.js: specs './tests/**/*.js'
+* Add in package.json: "test": "wdio ./wdio.conf.js"
```

Comment:
This is currently present in your wdio.conf.js file, so users don't have to do it

Reply:
done